packages/mdc-ripple/mdc-ripple.scss

```diff
@@ -17,13 +17,14 @@
 @import "@material/animation/functions";
 @import "@material/theme/mixins";
 @import "./mixins";
+@import "./variables";
 
 // postcss-bem-linter: define ripple-surface
 
 .mdc-ripple-surface {
-  @include mdc-ripple-base;
-  @include mdc-ripple-bg((pseudo: "::before"));
-  @include mdc-ripple-fg((pseudo: "::after"));
+  @include mdc-ripple-surface;
+  @include mdc-ripple-color;
```

Comment:
So you have to include mdc-ripple-color and mdc-ripple-radius even if your color is default and your radius is default? (Actually, is there a default radius?)

Reply:
Yeah, just like you had to always include -fg and -bg before.

And yes, there absolutely is a default radius (100%) - in fact, we _never_ customized this in our code prior to my realizing that checkbox and radio should have been doing so (they were manually overriding the exact styles involved rather than changing $radius in the ripple config).